package/root/usr/local/sbin/resize_rootfs.sh

```diff
@@ -5,24 +5,22 @@ if [[ "$(id -u)" -ne "0" ]]; then
 	exit 1
 fi
 
-case $(findmnt / -n -o SOURCE) in
-	/dev/mmcblk0p7)
```

Comment:
Can we use `case` with wildcard match instead?

```
case dev=$(findmnt / -n -o SOURCE) in
   /dev/mmcblk*)
     ;;
   /dev/sd*)
     ;
...
```

Reply:
either way you need to deal with the partition name differences, mmcblk0p7 v sda7 etc

Reply:
Yes, this is correct, but case is more readable than if :)